addons/website/views/snippets/s_banner.xml

```diff
@@ -4,7 +4,7 @@
 <template id="s_banner" name="Banner">
```

Comment:
Commit message:
- `prevents to choose the number` -> prevents from choosing :wink: 
- `that is present on its row as it prevents...` -> to be more clear, say `...that is present on the "Banner" snippet row, as it prevents from choosing its number of columns with the "Column Count" option` (so we directly now which row we are talking about, + missing comma :slightly_smiling_face: )